packages/backend/src/models/NotificationsModel/NotificationsModel.ts

```diff
@@ -0,0 +1,125 @@
+import { Notification } from '@lightdash/common';
+import { Knex } from 'knex';
+import {
+    DashboardTileCommentsTableName,
+    DbDashboardTileComments,
+} from '../../database/entities/comments';
+import { DashboardsTableName } from '../../database/entities/dashboards';
+import {
+    DbNotifications,
+    NotificationsTableName,
+} from '../../database/entities/notifications';
+import { UserTableName } from '../../database/entities/users';
+
+type NotificationsModelDependencies = {
+    database: Knex;
+};
+
+export class NotificationsModel {
+    private readonly database: Knex;
+
+    constructor(deps: NotificationsModelDependencies) {
+        this.database = deps.database;
+    }
+
+    async getDashboardCommentNotifications(
+        userUuid: string,
+    ): Promise<Notification[]> {
+        const notificationsWithUserAndDashboardAndAuthor: (DbNotifications &
+            Pick<DbDashboardTileComments, 'dashboard_tile_uuid'> & {
+                user_first_name: string;
+                user_last_name: string;
+                dashboard_name: string;
+                author_first_name: string;
+                author_last_name: string;
+            })[] = await this.database(NotificationsTableName)
+            .join(
+                UserTableName,
+                `${NotificationsTableName}.user_uuid`,
+                '=',
+                `${UserTableName}.user_uuid`,
+            )
+            .join(
+                DashboardTileCommentsTableName,
+                `${NotificationsTableName}.comment_id`,
+                '=',
+                `${DashboardTileCommentsTableName}.comment_id`,
+            )
+            .join(
+                `${UserTableName} as comment_authors`,
+                `${DashboardTileCommentsTableName}.user_uuid`,
+                '=',
+                'comment_authors.user_uuid',
+            )
+            .join(
+                DashboardsTableName,
+                `${NotificationsTableName}.dashboard_uuid`,
+                '=',
+                `${DashboardsTableName}.dashboard_uuid`,
+            )
+            .select(
+                `${NotificationsTableName}.*`,
+                `${UserTableName}.first_name as user_first_name`,
+                `${UserTableName}.last_name as user_last_name`,
+                `${DashboardsTableName}.name as dashboard_name`,
+                `${DashboardTileCommentsTableName}.dashboard_tile_uuid`,
+                'comment_authors.first_name as author_first_name',
+                'comment_authors.last_name as author_last_name',
+            )
+            .where(`${NotificationsTableName}.user_uuid`, userUuid)
+            .orderBy(`${NotificationsTableName}.created_at`, 'desc');
+
+        return notificationsWithUserAndDashboardAndAuthor.map((notif) => ({
+            notificationId: notif.notification_id,
+            user: {
+                name: `${notif.user_first_name} ${notif.user_last_name}`,
+            },
```

Comment:
I'm thinking that we don't need `user` yet. 
The `author` is important to display the notification like
`${author} has mentioned you`

Reply:
why don't we build the message in the backend (even store it) ? this will avoid passing  small details like the dashboard name,and simplify the changes in the future. 
I believe the whole message will be a link, so don't need to `structure` the message for the frontend...

Reply:
done! Here in `notificationsModel` https://github.com/lightdash/lightdash/pull/9131/commits/7315bf4c9ca5f00ea3dd48ca996f0d3e7e3059a5